pkg/rid/store/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package store provides the interface to obtain Repos, to interact with
```

Comment:
"Package store defines interfaces that every RID Store implementation, acting on an RID Repository, must implement"